.github/workflows/perf-pr.yml

```
@@ -34,17 +34,20 @@ concurrency:
 
 jobs:
   measure:
     name: Compare instruction counts
-    if: github.event.pull_request.user.login == 'jdx'
+    if: github.event.pull_request.user.login == 'jdx' && github.event.pull_request.head.repo.full_name == github.repository
     # Same runner class as perf.yml and perf-backfill.yml. Absolute counts shift
     # between machine types by more than a real regression does, so a comparison
     # across runner classes is not a comparison — tak refuses to make one, and
     # the report would say "nothing was compared" instead of anything useful.
-    runs-on: bamboo-perf
+    runs-on: [self-hosted, jdx-perf-v1]
```

actionlint (1.7.12)

[error] label "jdx-perf-v1" is unknown. available labels are "windows-latest", "windows-latest-8-cores", "windows-2025", "windows-2025-vs2026", "windows-2022", "windows-11-arm", "ubuntu-slim", "ubuntu-latest", "ubuntu-latest-4-cores", "ubuntu-latest-8-cores", "ubuntu-latest-16-cores", "ubuntu-24.04", "ubuntu-24.04-arm", "ubuntu-22.04", "ubuntu-22.04-arm", "macos-latest", "macos-latest-xlarge", "macos-latest-large", "macos-26-intel", "macos-26-xlarge", "macos-26-large", "macos-26", "macos-15-intel", "macos-15-xlarge", "macos-15-large", "macos-15", "macos-14-xlarge", "macos-14-large", "macos-14", "self-hosted", "x64", "arm", "arm64", "linux", "macos", "windows", "bamboo-perf", "macos-14", "namespace-profile-endev-macos-arm64", "namespace-profile-endev-macos-arm64-large", "namespace-profile-endev-macos-arm64;overrides.cache-tag=mise-pr-rust-macos", "namespace-profile-endev-macos-arm64-large;overrides.cache-tag=mise-release-rust-macos", "namespace-profile-endev-linux-amd64", "namespace-profile-endev-linux-amd64-large", "namespace-profile-endev-linux-amd64-xlarge", "namespace-profile-endev-linux-amd64;overrides.cache-tag=mise-pr-rust-linux", "namespace-profile-endev-linux-amd64;overrides.cache-tag=mise-pr-rust-linux-nightly", "namespace-profile-endev-linux-amd64;overrides.cache-tag=mise-release-plz-rust-linux", "namespace-profile-endev-linux-amd64-large;overrides.cache-tag=mise-pr-rust-linux", "namespace-profile-endev-linux-amd64-large;overrides.cache-tag=mise-perf-main", "namespace-profile-endev-linux-amd64-large;overrides.cache-tag=mise-release-rust-linux", "buildjet-32vcpu-ubuntu-2204-arm", "buildjet-16vcpu-ubuntu-2204-arm", "buildjet-8vcpu-ubuntu-2204-arm", "buildjet-4vcpu-ubuntu-2204-arm", "buildjet-2vcpu-ubuntu-2204-arm", "buildjet-32vcpu-ubuntu-2204", "buildjet-16vcpu-ubuntu-2204", "buildjet-8vcpu-ubuntu-2204", "buildjet-4vcpu-ubuntu-2204", "buildjet-2vcpu-ubuntu-2204". if it is a custom label for self-hosted runner, set list of labels in actionlint.yaml config file

(runner-label)